lib/gluegun.rb

```diff
@@ -9,6 +9,18 @@ module Gluegun
 
   class Gluegun
 
+    def self.raw(link)
```

Comment:
This can be a private function. There is no need to expose it and it can be called internally.